db/structure.sql

```diff
@@ -102,13 +102,21 @@ ALTER TABLE ONLY public.schema_migrations
     ADD CONSTRAINT schema_migrations_pkey PRIMARY KEY (version);
 
 
+--
+-- Name: pages_slug_idx; Type: INDEX; Schema: public; Owner: -
+--
+
+CREATE UNIQUE INDEX pages_slug_idx ON public.pages USING btree (lower(slug));
```

Comment:
yeah hash indexes (which are good for this same kind of query) would take up marginally less space but couldn't be used for other kinds of queries. from my [reading](https://www.citusdata.com/blog/2017/10/17/tour-of-postgres-index-types/) into this:

- use btree by default since it's good at most things
- use hash if you are on >v10 and only ever do equality comparison and need to save space
- use GIN for multiple values e.g. arrays/JSON/hstore
- use GiST for rows that overlap values e.g. geometry types/full text search
- SP-GiST/BRIN for larger data